test/managedcluster/resources/vsphere-standalone-cp.yaml.tpl

```
@@ -3,7 +3,7 @@ kind: ManagedCluster
 metadata:
   name: ${MANAGED_CLUSTER_NAME}
 spec:
-  template: vsphere-standalone-cp
+  template: vsphere-standalone-cp-vsphere
```

This looks like a find and replace mistake:

```suggestion
  template: vsphere-standalone-cp
```